prep-html-skeleton/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Skeleton Exercise </title>
```

Comment:
Hi Nick, be sure to include a closing `</head>` tag before your opening `<body>` tag.

Reply:
Noted. Do I resubmit the corrections onto GitHub doing the same uploading steps as before?